packages/account/src/Components/poi/status/unsupported/unsupported.jsx

```diff
@@ -19,12 +20,18 @@ const checkNimcStep = documents => {
 
 const Unsupported = ({ country_code, ...props }) => {
     const [detail, setDetail] = React.useState(null);
-    const toggleDetail = index => setDetail(index);
+    const toggleDetail = index => {
+        setDetail(index);
+    };
     const documents = getDocumentIndex({
         setDetail,
         country_code,
     });
 
+    if (props?.manual && props.manual.status === 'pending') {
+        return <UploadComplete is_from_external={true} needs_poa={false} />;
```

Comment:
believe that we can just pass `is_from_external` instead of `is_from_external={true}`
https://github.com/george-usynin-binary/deriv-app/blob/master/docs/JavaScript/README.md#:~:text=Do%20NOT%20use%20%3CElement%20attributeName%3D%7Btrue%7D%20/%3E%3B%20just%20use%20%3CElement%20attributeName%20/%3E.

Reply:
Will fix this